### ISSUE TITLE
Target variable prediction With time series: separate train and test set

### ISSUE THREAD

**Issue by Daisy8po** [philipperemy/keras-tcn#228]
I need to predict a target variabile using a tcn model based on time series.

Now i follow your code of milk production but i need to consider in the X array all the features of my dataset as predictors, instead on the y i need only one features, my target.
how can i solve this?

Furthermore i have indeed two separated dataset, one for train and one for test, so i need to create from they my couple of x and y train, and x and y test.

I'm struggle on this. Thank you very much

**Comment by philipperemy** [comment]
@Daisy8po I'm not sure exactly what you mean by that. Can you detail a bit more, maybe give a concrete example with numbers?

**Comment by philipperemy** [comment]
The TCN layer input/output work exactly like an LSTM. If your X is multivariable and your Y univariate, you're looking at a Many-To-One regression problem. You can google "Keras LSTM many to one regression". And in any tutorial you can swap the LSTM class by the TCN class and it should work seamlessly.
